addons/im_livechat/static/src/core/web/transcript_sender_popover.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="im_livechat.TranscriptSenderPopover">
+        <ActionPanel title.translate="Send a copy of the conversation" icon="'fa fa-paper-plane'" resizable="false">
+            <div class="input-group my-2 shadow-sm">
+                <input t-ref="autofocus" t-on-keydown="onKeydown" t-model="state.email" t-att-disabled="[STATUS.SENDING, STATUS.SENT].includes(state.status)" type="text" class="form-control bg-view" placeholder="mail@example.com" />
+                <button class="btn btn-primary" type="button" data-action="sendTranscript" t-att-disabled="isButtonDisabled" t-on-click="onClickSend">
+                    <i class="fa" t-att-class="{
+                        'fa-circle-o-notch fa-spin': state.status === STATUS.SENDING,
+                        'fa-check': state.status === STATUS.SENT,
+                        'fa-paper-plane': state.status === STATUS.IDLE,
+                        'fa-repeat': state.status === STATUS.FAILED,
+                    }" />
+                </button>
+            </div>
```

Comment:
It’s not okay to duplicate the entire template and logic of the transcript sender. The popover should simply be an action panel that embeds the transcript_sender component. Currently, we are duplicating the UI, button disabled condition, state, email validation, sending process/failure detection, and status enum.

However, the addition of the keydown handler is good and should be incorporated into the transcript sender component.